semana12/Aula3-Projeto-WeekPlanner/src/actions/index.test.js

```diff
@@ -0,0 +1,57 @@
+import {setTaskDayAction, getTasks, createTask} from '.'
+import axios from 'axios';
+
+const tasks = [{
+    id: 1,
+    text: 'helloWorld',
+    day: 'Segunda'
+}]
+
+describe('Teste das actions', () => {
+
+    it('teste para saber se o setTasks está retornando as tasks ', () => {
+        
+        const expectedAction = {
+            type: "SET_TASK_DAY_ACTION",
+            payload: {
+                task: tasks
+            }
+        }
+
+        const actions = setTaskDayAction(tasks)
+        
+        expect(actions).toEqual(expectedAction)
+    });
+});
+
+describe('teste do getTasks', () => {
+    it('teste para saber se o getTasks retorna um valor', async () => {
```

Comment:
Também não precisa explicitar que é um teste. Procure colocar essa descricao do `it` no seguinte formato: "deve retornar um valor". E o describe seria só "getTasks". Então, o describe define o que está sendo testado, e o it define o que exatamente cada teste testa.